build-aux/flathub-export.sh

```diff
@@ -1,16 +1,12 @@
 #!/bin/bash
-export DIST="export/flathub/iplan" &&
-rm -rf $DIST &&
-mkdir $DIST &&
-rsync -a --exclude-from='../.gitignore' --exclude='ir.imansalmani.IPlan.Devel.json' --exclude='ir.imansalmani.IPlan.json' --exclude='build-aux' ../ $DIST &&
-cp ir.imansalmani.IPlan.json $DIST/ &&
-cd $DIST &&
-mkdir .cargo &&
-touch .cargo/config &&
-cargo vendor | sed 's/^directory = ".*"/directory = "vendor"/g' > .cargo/config &&
-git init &&
-git add . &&
-git commit -m "Build commit" &&
-cd ../ &&
-tar -zcvf iplan.tar.gz iplan &&
-sha256sum iplan.tar.gz > checksum
+set -e
+
+DIST="./export/flathub/iplan"
+FLATHUB="$(dirname $DIST)"
+rm -rf $DIST
+mkdir -p $DIST
+rsync -a --exclude-from='../.gitignore' --exclude='ir.imansalmani.IPlan.Devel.json' --exclude='ir.imansalmani.IPlan.json' --exclude='build-aux' ../ $DIST
+cp ir.imansalmani.IPlan.json $DIST/
+cargo -q vendor
+tar -zcf iplan.tar.gz "$FLATHUB"/iplan
```

Comment:
`"$FLATHUB"/iplan` same as `$DIST`
Use this `tar -zcf "$FLATHUB"/iplan.tar.gz $DIST`